io/net/virt-net/lpm.py

```diff
@@ -227,7 +229,7 @@ def install_packages(self):
         detected_distro = distro.detect()
         self.log.info("Test is running on: %s", detected_distro.name)
         for pkg in ['ksh', 'src', 'rsct.basic', 'rsct.core.utils',
-                    'rsct.core', 'DynamicRM']:
+                    'rsct.core', 'DynamicRM', 'python3-pexpect']:
```

Comment:
The test would fail when trying to import the `pxssh` module. So this may not solve any problem. Or am I missing any other value add with this?

Reply:
User is expected to have pexpect installed before starting this test.